tests/GridFieldOrderableRowsTest.php

```diff
@@ -296,4 +301,41 @@ public function testReorderItemsSubclassVersioned()
 
         $this->assertTrue($differenceFound);
     }
+
+    public function testGetManipulatedDataWithoutDefaultSort()
+    {
+        $sortedList = $this->getTitleSortedListForManipuatedData(TitleObject::class, [
+            ['Title' => 'C'],
+            ['Title' => 'A'],
+            ['Title' => 'B'],
+        ]);
+        $this->assertSame(['A', 'B', 'C'], $sortedList->column('Title'));
+    }
+
+    public function testGetManipulatedDataWithDefaultSort()
+    {
+        $sortedList = $this->getTitleSortedListForManipuatedData(TitleSortedObject::class, [
+            ['Title' => 'Z', 'Iden' => 'C', 'DefaultSort' => 3],
+            ['Title' => 'Z', 'Iden' => 'A', 'DefaultSort' => 2],
+            ['Title' => 'Z', 'Iden' => 'B', 'DefaultSort' => 1],
```

Comment:
Why are we using `Iden` here instead of `Title`? It seems like it would be a lot clearer if we use `Title` like with the above test, so we can clearly contrast the results of _this_ sorting vs the sorting by title that happens above?

Reply:
This is an old PR so I cannot remember exact reasoning

Best guess is that in `getTitleSortedListForManipuatedData()` we have `$orderable = new GridFieldOrderableRows('Title');`, and I wanted to test what happens with `Title` is the same value, so it falls back to `$default_sort`

Reply:
Fair enough.